unionfs/autounion_test.go

```diff
@@ -142,14 +144,22 @@ func TestAutoFsSymlink(t *testing.T) {
 		t.Error("error writing:", err)
 	}
 
-	fi, _ = os.Lstat(wd + "/mnt/manual1")
-	if fi != nil {
-		t.Error("Should not have file:", fi)
-	}
-
-	_, err = ioutil.ReadDir(wd + "/mnt/config")
-	if err != nil {
-		t.Fatalf("ReadDir failed: %v", err)
+	// Note that FUSE API 7.11 introduced the notification support required to cause the node to be immediately removed. This check will fail in earlier versions.
+	if _, minor := server.Version(); minor >= 11 {
```

Comment:
It's all keyed on the InitIn data, though, so I think it should rather be on InitIn.